pkg/kubelet/cm/devicemanager/manager_test.go

```diff
@@ -86,8 +103,8 @@ func TestDevicePluginReRegistration(t *testing.T) {
 	devices := m.Devices()
 	require.Equal(t, 2, len(devices[testResourceName]), "Devices are not updated.")
```

Comment:
Hope you wont mind adding couple of another minor changes as well. Can you please replace all `<-callbackChan` with : 
```golang
	select {
 	case <-callbackChan:
 		break
 	case <-time.After(time.Second):
 		t.FailNow()
 }
```
```